### PR TITLE
Dagger

### PR DESCRIPTION
**Pull request by chanacham**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,6 +10,24 @@ repositories {
...
+//    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
+//    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+//
+//    implementation 'com.google.code.gson:gson:2.10.1'
+//    testImplementation 'org.mockito:mockito-core:5.1.1'
+//
+//
+//    implementation 'com.squareup.retrofit2:retrofit:(2.9.0)'
+//    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
+//    implementation 'com.squareup.retrofit2:adapter-rxjava3:2.9.0'
```
Comment (andrewoid) [inline build.gradle]: Remove this commented out stuff

File: src/main/java/weather/current/CurrentWeather.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import weather.sharedclasses.Clouds;
+import weather.sharedclasses.Coord;
+import weather.sharedclasses.Weather;
+import weather.sharedclasses.Wind;
+
+import java.util.List;
+
+public class CurrentWeather {
+
+    private String mBase;
```
Comment (andrewoid) [inline src/main/java/weather/current/CurrentWeather.java]: As discussed, you can't have "m" before your member variables in json classes.

File: src/main/java/weather/sharedclasses/Clouds.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+package weather.sharedclasses;
+
+public class Clouds {
+
+    private Long mAll;
```
Comment (andrewoid) [inline src/main/java/weather/sharedclasses/Clouds.java:6]: You still have m before some variable names.

**Review by andrewoid — APPROVED** [review]

**Comment by andrewoid** [comment]
Fix your failing test.

**Review by andrewoid — CHANGES_REQUESTED** [review]
I'll approve after you fix it.

File: src/main/java/weather/sharedclasses/Clouds.java (hunk 3 above)
Comment (chanacham) [reply src/main/java/weather/sharedclasses/Clouds.java:6]: Fixed

**Review by andrewoid — APPROVED** [review]

**Pull request merged**